packages/app/src/cli/models/extensions/specifications/payments_app_extension_schemas/credit_card_payments_app_extension_schema.ts

```diff
@@ -29,6 +29,7 @@ export const CreditCardPaymentsAppExtensionSchema = BasePaymentsAppExtensionSche
     targeting: zod.array(zod.object({target: zod.literal(CREDIT_CARD_TARGET)})).length(1),
     verification_session_url: zod.string().url().optional(),
     ui_extension_handle: zod.string().optional(),
+    supports_moto: zod.boolean(),
```

Comment:
Is this is not optional, it will break any existing extension that doesn't have it. Is this an accepted outcome? should it have a default value?

Reply:
When I say existing extensions, i mean at the moment of updating them via CLI, it will fail until they provide this value.

Simple case:
- You have an automated deploy in CI, if we release this, that deploy will start failing until you provide this value.

Is this the expected behaviour? If it is, then OK; just wanted to confirm!

Reply:
Thanks. I think it's worth it to provide default values similar to what is done in core